gateway/src/resty/file.lua

```diff
@@ -0,0 +1,32 @@
+local co_yield = coroutine.yield
```

Comment:
OpenResty has own version of `yield` https://github.com/openresty/lua-resty-core/commit/97b5825929ee4860b4a54b3905e2e191fc61f41e
Would be good to investigate and see what is the one we should use.

The API is injected in https://github.com/openresty/lua-nginx-module/blob/2a190736a58674086c3a27bf71a7993383fffb55/src/ngx_http_lua_coroutine.c#L288

Also since https://github.com/openresty/lua-nginx-module/pull/1239 is in it might be worth looking at how it affects us.

Also https://github.com/Tieske/Penlight/issues/265#issuecomment-423576100 is quite true :)